docs/app/Examples/modules/Checkbox/Variations/Variations.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample';
+
+export default class extends Component {
+  render() {
+    return (
+      <div>
+        <ComponentExample
+          title='Fitted'
+          description='A fitted checkbox does not leave padding for a label'
+          examplePath='modules/Checkbox/Variations/Fitted'
+        />
+      </div>
```

Comment:
Extra `div` not needed here, only when nesting multiple children.  More accurately, a component can only have a single root node with no siblings.

If we adopt the [above comment](#r42073397) regarding `ExampleSection` wrapping `ComponentExamples` this comment will no longer be valid.